go/cmd/vtctldclient/internal/command/vschemas.go

```diff
@@ -33,8 +39,117 @@ var (
 		Args: cobra.ExactArgs(1),
 		RunE: commandGetVSchema,
 	}
+	// ApplyVSchema makes an ApplyVSchema gRPC call to a vtctld.
+	ApplyVSchema = &cobra.Command{
+		Use:  "ApplyVSchema {-vschema=<vschema> || -vschema_file=<vschema file> || -sql=<sql> || -sql_file=<sql file>} [-cells=c1,c2,...] [-skip_rebuild] [-dry-run] <keyspace>",
+		Args: cobra.ExactArgs(1),
+		RunE: commandApplyVSchema,
+	}
 )
 
+var applyVSchemaOptions = struct {
+	VSchema     string
+	VSchemaFile string
+	SQL         string
+	SQLFile     string
+	DryRun      bool
+	SkipRebuild bool
+	Cells       []string
+}{}
+
+func commandApplyVSchema(cmd *cobra.Command, args []string) error {
+	keyspace := cmd.Flags().Arg(0) // validated on the server-side
```

Comment:
happy to validate here as well. wdyt?

Reply:
imo we should validate as much server-side as possible, that way the implementation can be shared between vtctldserver and the legacy vtctl commands